### ISSUE TITLE
Implement all rules

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/standard-readme-cli#4]
Currently, this linter does not catch all of the rules in standard-readme. This is because we have not created all of the rules needed to match the spec in [standard-readme](https://github.com/RichardLitt/standard-readme). These rules will be added to [standard-readme-preset](https://github.com/RichardLitt/standard-readme-preset). 

This is the official tracking issue for these rules.

**Comment by lukasoppermann** [comment]
Hey, how is this going? Currently only the headline is throwing an error.

**Comment by RichardLitt** [comment]
@lukasoppermann Yep. There haven't been any pushes yet; I need to have some more development time, but I'm currently on a semi-holiday in Scotland and do not have a lot of time for coding. Early June will be more productive. :) If you'd like to jump in and make some rules, feel free! :D

**Comment by lukasoppermann** [comment]
Jesus, I just looked through the chain required to add a rule. That is a lot of work. I thought they were already done in the spec. Sorry not able to help with this atm. But I understand why this takes a while. I am already adding standard-readme to every new package I build. 😉 

Anyway, enjoy your holiday.

**Comment by RichardLitt** [comment]
Yep. It'll be some work. That's OK! It'll be worth it.

**Comment by wooorm** [comment]
@lukasoppermann Before any programming starts, we first need a list of rules that can be implemented based on [the spec](https://github.com/RichardLitt/standard-readme/blob/master/spec.md).

* `filename`: should be `README.md` or `README.$bcp47.md`; `readme.md`, `Readme.md`, `README.mkd`, and the like are invalid.
* ...

So maybe you’d be able to help with that?

**Comment by RichardLitt** [comment]
That's right, @wooorm! Sorry for the delay. Traveling. Great meeting you in Berlin. :)

**Comment by wooorm** [comment]
Yes, definitely! And no problem at all, enjoy your semi-holiday!

**Comment by piranna** [comment]
Seems [standard-readme-preset](https://github.com/RichardLitt/standard-readme-preset) has got some love lately, can this module be updated to match with latest version? :-)

**Comment by RichardLitt** [comment]
It's got a lot of work which needs to be done. :) Happy to look at PRs.

**Comment by piranna** [comment]
Any update on this? :-)

**Comment by RichardLitt** [comment]
No plans at the moment. Focusing on other things. PRs accepted.
